src/components/PersonLink.tsx

```diff
@@ -0,0 +1,87 @@
+import { FC } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../types';
+
+type Props = {
+  person: Person,
+  isSelected: boolean,
+};
+
+export const PersonLink: FC<Props> = ({
+  person,
+  isSelected,
+}) => {
+  const location = useLocation();
+  const isPepoleLocation = location.pathname === '/people';
+  const {
+    name,
+    sex,
+    born,
+    died,
+    motherName,
+    mother,
+    fatherName,
+    father,
+    slug,
+  } = person;
+
+  const parent = (
```

Comment:
Consider renaming 'parent' function to be more descriptive, such as 'renderParentLink'.